azure/configure.yml

```diff
@@ -61,7 +61,6 @@ steps:
         --with-imap \
         --with-kerberos \
         --with-imap-ssl \
-        --enable-werror \
```

Comment:
Drop these now, or is there still something left?

Reply:
There is one remaining failure on 32bits:
```
/home/vsts/work/1/s/ext/sodium/libsodium.c: In function ‘zif_sodium_crypto_secretstream_xchacha20poly1305_push’:
/home/vsts/work/1/s/ext/sodium/libsodium.c:3181:71: warning: logical ‘or’ of equal expressions [-Wlogical-op]
 3181 |  if (msg_len > crypto_secretstream_xchacha20poly1305_MESSAGEBYTES_MAX ||
      |     
```